.github/workflows/publish_to_pypi.yml

```
@@ -0,0 +1,116 @@
+    - name: Download all the dists
+      uses: actions/download-artifact@v4
+      with:
+        name: python-package-distributions
+        path: dist/
+    - name: Verify downloaded artifacts
+      run: |
+        ls dist/*.whl dist/*.tar.gz || exit 1
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        retry-on-error: true
```

actionlint

input "retry-on-error" is not defined in action "pypa/gh-action-pypi-publish@release/v1". available inputs are "attestations", "packages-dir", "packages_dir", "password", "print-hash", "print_hash", "repository-url", "repository_url", "skip-existing", "skip_existing", "user", "verbose", "verify-metadata", "verify_metadata"

(action)